codegen/src/test/resources/q4/Q4Aggregate2ProcessFunction.scala

```diff
@@ -0,0 +1,19 @@
+import org.hkust.RelationType.Payload
+import org.apache.flink.api.common.state.ValueStateDescriptor
+import org.apache.flink.api.common.typeinfo.{TypeHint, TypeInformation}
+import org.hkust.BasedProcessFunctions.AggregateProcessFunction
+class Q4Aggregate2ProcessFunction extends AggregateProcessFunction[Any, Integer]("Q4Aggregate2ProcessFunction", Array("O_ORDERPRIORITY"), Array("O_ORDERPRIORITY"), aggregateName = "order_count", deltaOutput = true) {
+   override def aggregate(value: Payload): Integer = {
+      if(value("DIST_COUNT").asInstanceOf[String].toInt==1) 1
+if(value("DIST_COUNT").asInstanceOf[String].toInt==0) -1
```

Comment:
Line 8 should have "else" at the beginning, makes the query an if-else { if-else } query.